src/main.cpp

```diff
@@ -2244,7 +2244,10 @@ void vcRenderWindow(vcState *pProgramState)
     {
       if (ImGui::Begin(udTempStr("%s###sceneExplorerDock", vcString::Get("sceneExplorerTitle")), &pProgramState->settings.window.windowsOpen[vcDocks_SceneExplorer]))
       {
-        if (vcMenuBarButton(pProgramState->pUITexture, vcString::Get("sceneExplorerAddUDS"), vcString::Get("sceneExplorerAddUDSKey"), vcMBBI_AddPointCloud, vcMBBG_FirstItem) || vcHotkey::IsPressed(vcB_AddUDS))
+        char buffer[50] = {};
+        vcHotkey::GetKeyName(vcB_AddUDS, buffer, udLengthOf(buffer));
```

Comment:
It might be better to add a template overload similar to how `udLengthOf` works so that you can just call `vcHotkey::GetKeyName(vcB_AddUDS, buffer)`. This reduces the chances of the two parameters getting out of sync which has happened before.